cristin-project/src/test/java/no/unit/nva/cristin/projects/create/createProject.feature

```diff
@@ -10,11 +10,11 @@ Feature: API tests for Cristin Project retrieve and search
     * def token = tokenGenerator.loginUser(username, password, cognitoClientAppId)
-    * def minimalCreateRequest = { type: 'Project', title: 'Test Project', language: 'http://lexvo.org/id/iso639-3/nob', startDate: '2010-06-29T01:33:17.518Z',  coordinatingInstitution: {type: 'Organization',id: 'https://api.cristin-test.uio.no/v2/institutions/20202'},contributors: [{type: 'ProjectManager',identity: {type: 'Person',id: 'https://api.cristin-test.uio.no/v2/persons/515114'},affiliation: {type: 'Organization',id: 'https://api.cristin-test.uio.no/v2/institutions/20202'}}]}
-    * def minimalCreateRequestWithUnitIdentifiers =   { type: 'Project', title: 'Test Project', language: 'http://lexvo.org/id/iso639-3/nob', startDate: '2010-06-29T01:33:17.518Z',  coordinatingInstitution: {type: 'Organization',id: 'https://api.dev.nva.aws.unit.no/cristin/organization/20202.0.0.0'},contributors: [{type: 'ProjectManager',identity: {type: 'Person',id: 'https://api.dev.nva.aws.unit.no/cristin/person/515114'}}]}
-    * def lessThanMinimalCreateRequest = { type: 'Project', startDate: '2010-06-29T01:33:17.518Z',  coordinatingInstitution: {type: 'Organization',id: 'https://api.cristin-test.uio.no/v2/institutions/20202'},contributors: [{type: 'ProjectManager',identity: {type: 'Person',id: 'https://api.cristin-test.uio.no/v2/persons/515114'},affiliation: {type: 'Organization',id: 'https://api.cristin-test.uio.no/v2/institutions/20202'}}]}
-    * def requestWithIllegalValues =   { type: 'Project', startDate: '2010-06-29T01:33:17.518Z',  coordinatingInstitution: {type: 'Organization',id: 'https://api.cristin-test.uio.no/v2/institutions/20202'},contributors: [{type: 'ProjectMan',identity: {type: 'Person',id: 'https://api.cristin-test.uio.no/v2/persons/515114'},affiliation: {type: 'Organization',id: 'https://api.cristin-test.uio.no/v2/institutions/20202'}}]}
-    * def requestWithId =   { id: 'https://api.dev.nva.aws.unit.no/cristin/project/2121331275', type: 'Project', title: 'Test Project', language: 'http://lexvo.org/id/iso639-3/nob', startDate: '2010-06-29T01:33:17.518Z',  coordinatingInstitution: {type: 'Organization',id: 'https://api.cristin-test.uio.no/v2/institutions/20202'},contributors: [{type: 'ProjectManager',identity: {type: 'Person',id: 'https://api.cristin-test.uio.no/v2/persons/515114'},affiliation: {type: 'Organization',id: 'https://api.cristin-test.uio.no/v2/institutions/20202'}}]}
+    * def minimalCreateRequest = { type: 'Project', title: 'Test Project', language: 'http://lexvo.org/id/iso639-3/nob', startDate: '2010-06-29T01:33:17.518Z', endDate: '2030-06-29T01:33:17.518Z',  coordinatingInstitution: {type: 'Organization',id: 'https://api.cristin-test.uio.no/v2/institutions/20202'},contributors: [{type: 'ProjectManager',identity: {type: 'Person',id: 'https://api.cristin-test.uio.no/v2/persons/515114'},affiliation: {type: 'Organization',id: 'https://api.cristin-test.uio.no/v2/institutions/20202'}}]}
```

Comment:
What happens when enddate is passed and this test still runs?

Reply:
then the project gets status concluded in cristin